package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "api3",
-  "version": "0.0.1",
+  "version": "0.1.0",
```

Comment:
Is there any way we could manage the version from a single place like `lerna.json`? Right now, there are loads of places with 0.1.0 as the version and it's going to be easy to miss updating one

Reply:
I don't think there is but it's not really a problem. The publishing will be done with Lerna and it will take care of updating all the versions for us :slightly_smiling_face: